docs/composer-core/business-capability.md

```diff
@@ -58,24 +74,22 @@ Coming soon.
 
 ## Edit Provider Mapping
 
-:::info
-
-Coming soon.
-
-:::
-
-See our [JSONata reference](../reference/jsonata-reference.md) for examples.
+1. Open the business capability whose mappings you want to edit.
+2. Under "Mappings", click on the mapping you want to change.
+3. A new page with a full screen editor should open. There you can use JSONata to map the source system properties to your business capability properties. See our [JSONata reference](../reference/jsonata-reference.md) for examples.
+    1. If it's an action business capability, select "Request" or "Response" from the headline to add the respective mapping.
 
 ## Delete Provider Mapping
 
 1. Open the business capability that you want to remove the provider mapping from.
-2. Under "Mappings" click on the mapping you wish to delete
-3. Use the menu on the top-right and click on "Delete"
+2. Under "Mappings" click on the mapping you wish to delete.
+3. Use the menu on the top-right and click on "Delete".
 
 ## Refresh Business Capability Data
 
-:::info
+At the moment, we don't have any automatic refresh interval or webhooks set up. That means, to get the latest data from your source systems, you need to manually refresh the index.
```

Comment:
nitpicky English comment here: none of these commas are necessary

Reply:
oh i've had this PR open for so long i didn't even see that it was merged before i posted this comment - sorry

Reply:
Ha, no worries, I'll try and remember to fix this with the next batch of updates. 

Commas are my nemesis.